Password LDAP seems not working

Hello,

When I note ldap_search_password and I check connectivity, I get this log in Harbor :
2021-03-05T12:35:44Z [ERROR] [/lib/http/error.go:54]: {"errors":[{"code":"INTERNAL_SERVER_ERROR","message":"LDAP connect fail, error: LDAP Result Code 206 \"**Empty password not allowed by the client**\": ldap: empty password not allowed by the client"}]}

- Version 1.2.2

--- a/client/config.go
+++ b/client/config.go
@@ -43,6 +43,7 @@ func GetConfigAuth(d *schema.ResourceData) models.ConfigBodyPost {
 			AuthMode:     "ldap_auth",
 			LdapURL:      d.Get("ldap_url").(string),
 			LdapSearchDn: d.Get("ldap_search_dn").(string),
+			LdapSearchPassword: d.Get("ldap_search_password").(string),
 			LdapBaseDn:   d.Get("ldap_base_dn").(string),
 			LdapFilter:   d.Get("ldap_filter").(string),
 			LdapUID:      d.Get("ldap_uid").(string),

--- a/models/config.go
+++ b/models/config.go
@@ -32,6 +32,7 @@ type ConfigBodyPost struct {
 	LdapGroupBaseDn                 string `json:"ldap_group_base_dn,omitempty"`
 	LdapGroupAttributeName          string `json:"ldap_group_attribute_name,omitempty"`
 	LdapGroupMembershipAttribute    string `json:"ldap_group_membership_attribute,omitempty"`
+	LdapSearchPassword              string `json:"ldap_search_password,omitempty"`
 	EmailInsecure                   bool   `json:"email_insecure"`
 	LdapGroupAdminDn                string `json:"ldap_group_admin_dn,omitempty"`
 	EmailUsername                   string `json:"email_username,omitempty"`

--- a/provider/resource_config_auth.go
+++ b/provider/resource_config_auth.go
@@ -79,7 +79,7 @@ func resourceConfigAuth() *schema.Resource {
 				Type:          schema.TypeString,
 				Optional:      true,
 				ConflictsWith: ldapConflictsWith(),
-				RequiredWith:  ldapRequiredWith(),
+				RequiredWith:  ldapRequiredWith(),				
 			},
 			"ldap_uid": {
 				Type:          schema.TypeString,

--- a/provider/resource_garbage_collection_test.go
+++ b/provider/resource_garbage_collection_test.go
@@ -45,7 +45,7 @@ func TestAccGCUpdate(t *testing.T) {
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckResourceExists(harborGCMain),
 					resource.TestCheckResourceAttr(
-						harborGCMain, "schedule", "daily"),
+						harborGCMain, "schedule", "Daily"),
 				),
 			},
 			{
@@ -53,7 +53,7 @@ func TestAccGCUpdate(t *testing.T) {
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckResourceExists(harborGCMain),
 					resource.TestCheckResourceAttr(
-						harborGCMain, "schedule", "hourly"),
+						harborGCMain, "schedule", "Hourly"),
 					resource.TestCheckResourceAttr(
 						harborGCMain, "delete_untagged", "true"),
 				),
@@ -65,7 +65,7 @@ func TestAccGCUpdate(t *testing.T) {
 func testAccCheckGCBasic() string {
 	return fmt.Sprintf(`
 	resource "harbor_garbage_collection" "main" {
-		schedule        = "daily"
+		schedule        = "Daily"
 	}
 	`)
 }
@@ -73,7 +73,7 @@ func testAccCheckGCBasic() string {
 func testAccCheckGCUpdate() string {
 	return fmt.Sprintf(`
 	resource "harbor_garbage_collection" "main" {
-		schedule        = "hourly"
+		schedule        = "Hourly"
 		delete_untagged = true
 	}
 	`)

